src/js/player.js

```diff
@@ -2385,14 +2357,7 @@ class Player extends Component {
    *         The current source object
    */
   currentSource() {
-    const source = {};
-    const src = this.currentSrc();
```

Comment:
This code didn't make sense because `this.currentSrc` was looking at the same variable (`this.cache_.source`).